src/ontology/mondo-edit.obo

```diff
@@ -617799,6 +617814,83 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
 
+[Term]
+id: MONDO:1040083
+name: KIAA1549-related retinopathy
+def: "Any retinopathy caused by variants in the KIAA1549 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "KIAA1549-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/22219 ! KIAA1549
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040084
+name: NDP-related vitreoretinopathy including Norrie syndrome
```

Comment:
Is that really the requested term name? Why do they want to add "including Norrie syndrome"? it does not follow good term name best practice.